tests/test_setfit_model.py

```diff
@@ -0,0 +1,42 @@
+from sentence_transformers import SentenceTransformer
```

Comment:
I suggest moving these tests into `test_modeling.py` - it makes it a bit easier to find tests that are grouped by filename this way